libraries/botframework-schema/src/index.ts

```diff
@@ -2247,23 +2247,48 @@ export enum SemanticActionStateTypes {
 export enum Channels {
     Alexa = 'alexa',
     Console = 'console',
+    /**
+     * @deprecated This channel is no longer available for bot developers.
+     */
+    Cortana = 'cortana',
     Directline = 'directline',
     DirectlineSpeech = 'directlinespeech',
     Email = 'email',
     Emulator = 'emulator',
+    /**
+     * @deprecated This channel is no longer available for bot developers.
+     */
+    EnterpriseChannel = 'enterprisechannel',
     Facebook = 'facebook',
     Groupme = 'groupme',
+    /**
+     * @deprecated This channel is no longer available for bot developers.
+     */
+    Kaizala = 'kaizala',
```

Comment:
I don't think we should add a channel just to say it's deprecated. Please remove this one.

Reply:
Done. Thanks!